DISCOURSE.md

```diff
@@ -0,0 +1,45 @@
+# How to run loginbot for Discourse?
+
+The loginbot implements the OAuth2 specification partially and it covers the API [Discourse](https://www.discourse.org/) needs. Therefore, you can use it for 
+adding "Login with DeltaChat" to your Discourse instance. 
+
+## Getting a compiled version
+
+Each release comes with a compiled version for Linux x86 64 musl. This binary is stripped, optimized and should run on any Linux server with the
+said architecture regardless of the `libc` used in the distribution as the binary comes with its own `libc`. However, if you want it for some other 
+platform, you can [install the Rust toolchain](https://www.rust-lang.org/learn/get-started) and then `cargo build -r` in the project's directory
+to get an optimized release binary for your platform.
+
+## Prerequirements
+
+ - A server to run loginbot on it. Root access is NOT required nor it is recommended to run loginbot as root.
+ - A webserver like nginx to act as reverse proxy so that the outside world can talk with loginbot web APIs.
+ - An email account for the loginbot
+ - Admin access to the Discourse instance
+
+## Steps
+
+The first step assumes you are on Linux and you are fine with a statically linked binary with musl libc.
+
+ 1. Download the latest release from Releases and get the binary: `wget https://github.com/deltachat-bot/deltachat-loginbot/releases/download/v0.3.0/deltachat-loginbot-x86_64-linux-musl_0.3.0.tar.bz2`
+ 2. Extract loginbot's binary and other stuff to an empty directory: `tar xvf deltachat-loginbot-x86_64-linux-musl_0.3.0.tar.bz2 -C empty_directory`
+ 3. Rename `example_config.toml` to `config.toml` and modify the TOML config file(`example_config.toml`) to your need. Enter email address and password respectively in `email` and `password` fields in `config.toml`
```

Comment:
```suggestion
 3. Rename `example_config.toml` to `config.toml` and modify the TOML config file(`config.toml`) to your need. Enter the email address and password for the loginbot account into the `email` and `password` fields in `config.toml`
```

After the rename, the config.toml needs to be modified ;)
And the other suggestion is to clarify which email credentials need to go there.